FILE packages/website/src/components/landing-page.tsx
RANGE {"path":"packages/website/src/components/landing-page.tsx","old_lines":[1192,1196],"new_lines":[1192,1196],"diff_block_lines":[5,11],"hunk_header":"@@ -1192,5 +1192,5 @@ function SponsorCTA() {","substantive_changed_lines":2}
@@ -1192,5 +1192,5 @@ function SponsorCTA() {
         </p>
         <p>
-          If you like Paseo, sponsoring keeps development going.
+          If you like Paseo, consider sponsoring development.
         </p>
         <p>- Mo</p>
